docs/capabilities.md

```diff
@@ -4,33 +4,103 @@ time.  Whether you are working with large legacy codebases, your own
 Atomist provides you with the tools to speed development, smooth
 operations, and close the code/runtime feedback loop.
 
-Atomist helps you develop better software more quickly in several
-important ways:
-
--   Linking related events from source code repositories, continuous
-    integration, testing, and runtime environments, presenting them in
-    a unified manner to appropriate team members to greatly reduce
-    context-switching costs and incident time-to-resolution.
--   Managing a service life cycle, from commits to CI builds to
-    deployments to alerts to corrective action
--   Automating common tasks in software development and operation,
-    for example:
-    -   Using Rug editors to modify code in a consistent mannger
-        across many repositories
-    -   Ensuring a project conforms with your standards using a Rug
-        reviewer
-
-# What you can do with Atomist
-## Rug is the eventing runtime that ties everything together
+## Bringing everything together
+
+Atomist consumes events from the platforms and tools you use every day.
+By linking related events from source code repositories, continuous
+integration, testing, and runtime environments, Atomist can present
+them in a unified manner to appropriate team members via direct or
+group messages.  The result is massive reductions in context-switching
+costs and and shorter incident time-to-resolution.
+
+<!-- *That rug really tied the room together, did it not?* -->
+
+Rug is the eventing runtime that ties everything together.  Rug not
```

Comment:
I'd simply say "Rug is the runtime that ties everything together." Eventing is part of that, but not for everything in Rug. Also think this would make a good highlighted note or some other Admonition to make it stand out.